### PR TITLE
Adding license mention

### PR DESCRIPTION
**Pull request by billmcilhargey**
Mentioning license for this, which basically mentions this is included as part of owning M365

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/enterprise/office-365-network-mac-perf-overview.md [hunk 1]
```diff
@@ -197,6 +197,10 @@ An optimal Microsoft 365 service front door is one that is closest to your netwo
 
 The internet egress location is the location where your network traffic exits your enterprise network and connects to the Internet. This is also identified as the location where you have a Network Address Translation (NAT) device and usually where you connect with an Internet Service Provider (ISP). If you see a long distance between your location and your internet egress location, then this may indicate a significant WAN backhaul.
 
+### What license is needed for this capability?
+
+A license that provides access to the M365 Admin Center.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/office-365-network-mac-perf-overview.md]:
```suggestion
You require a  license that provides access to the Microsoft 365 admin center.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@computeronix

**Comment by yogkumgit** [comment]
@computeronix Gentle reminder

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**